### PR TITLE
Rational numbers - add test for reduce function - switch minus to numerator

### PR DESCRIPTION
**Pull request by kszambelanczyk**
Additional test that gives a clue about implementing minus shifting from denominator to numerator.

### CODE REVIEW

File: exercises/practice/rational-numbers/test/rational_numbers_test.exs [hunk 1]
```diff
@@ -308,5 +308,12 @@ defmodule RationalNumbersTest do
       result = {1, 1}
       assert RationalNumbers.reduce(r) == result
     end
+
+    @tag :pending
+    test "Reduce shift the minus sign on the numerator" do
```
Comment (jiegillet) [inline exercises/practice/rational-numbers/test/rational_numbers_test.exs:313]:
```suggestion
    test "Reduce places the minus sign on the numerator" do
```

**Review by jiegillet — APPROVED** [review]
Thank you, that's a nice addition. I suggested a small tweak in the test description.

**Comment by jiegillet** [comment]
Also, would you like to add your name in the list of contributors in `.meta/config.json`?

**Comment by jiegillet** [comment]
And the CI is not happy about formatting. Please run `mix format` in the test file.

**Review by angelikatyborska — APPROVED** [review]

**Comment by jiegillet** [comment]
I think this is ready to merge, I'm just waiting for your response on if you want to be added to the contributor's list. You are certainly entitled to.

**Comment by kszambelanczyk** [comment]
Oh, sorry I did not notice the question about the contributors list. Of course, I will be happy to. Thanks

**Comment by SleeplessByte** [comment]
This might also be nice to add to problem-specifications 😉🤗

**Pull request merged**